nightingaleproject/vrdr-dotnet
# Reorder death record properties to reflect order in standard death cert

This PR addresses [this ticket](https://boards.mitre.org/b/oaY5CRMEqJK3SsLrP/cdc-nvss/hu6PRoMNHZ45EH5sL)
Changed the priority values for death record properties to reflect the structure of the [US standard death certificate](https://www.cdc.gov/nchs/data/dvs/DEATH11-03final-ACC.pdf) 

To test
- Pull the updates and publish
- In the canary project, update the canary.csproj file to use the updated local vrdr-dotnet code
- Run canary and check the producing FHIR records and consuming FHIR records tools to see the updated field order and compare to the US standard death certificate
